src/rudderstack.ts

```diff
@@ -98,6 +98,10 @@ type TEvents = {
   identify: IdentifyAction;
 };
 
+type TTrackOptions = {
+  is_anonymous: boolean;
```

Comment:
I believe we have to add real id as well here because it's resolve only first half of problem
probably we can use it like `has_id`

Reply:
@NikitK-deriv this is for our `virtual_signup_flow` the point is, if user has the id he won't be signing up again

<img width="1728" alt="image" src="https://github.com/binary-com/deriv-analytics/assets/129206554/bde48682-ceb7-4461-99f0-c2f0ce5253a6">

we don't show the signup button once the user is signed in, so a user coming to signup if he is already logged in is a highly unlikely event, but still can you tell me why we need `has_id` ?